models/Challenge.js

```diff
@@ -1,29 +1,94 @@
-const queries = require('./queries');
 const utils = require('../utils');
 
 class Challenge {
-  makeChallenge(userId, spotId) {
-    const sql = queries.makeChallenge;
+  getAllChallengeablePeopleForUser(user) {
+    const sql = 'SELECT * FROM spots AS S, users AS U WHERE u.instrument = $1 AND u.part = $2 AND u.eligible = FALSE AND S.id = U.spotId';
```

Comment:
Then we're going to want to check if the spot has been challenged and add some styling it it is